website/src/libs/api-v3/amazonPayments.js

```diff
@@ -70,24 +46,15 @@ api.authorize = (inputSet) => {
   });
 };
 
-api.closeOrderReference = (inputSet) => {
-  let amzPayment = connect(amazonPayments);
-  return new Promize((resolve, reject) => {
-    amzPayment.offAmazonPayments.closeOrderReference(inputSet, (err, response) => {
-      if (err) return reject(err);
-      return resolve(response);
-    });
-  });
-};
-
-api.executePayment = (inputSet) => {
```

Comment:
So, that's part of the checkout route, that object isn't exported, it's labeled (for readability) and executed as async.series. We're getting rid of async, right? that `executePayment()` function finds user, does something with gifts, and calls `payments[method](data ... ` the equivalent is here: https://github.com/piousbox/habitrpg/blob/v3-payments-6/website/src/controllers/top-level/payments/amazon.js#L141 and the label became a comment for readability: https://github.com/piousbox/habitrpg/blob/v3-payments-6/website/src/controllers/top-level/payments/amazon.js#L132